lib/workers/repository/finalize/index.ts

```diff
@@ -39,3 +37,11 @@ export async function finalizeRepo(
   runBranchSummary(config);
   runRenovateRepoStats(config, prList);
 }
+
+// istanbul ignore next
```

Comment:
why ignore?

Reply:
the calling `finalizeRepo` function is ignored

https://github.com/renovatebot/renovate/blob/0ec88ed4352aaaa673969bb34ebf0e94edc97108/lib/workers/repository/finalize/index.ts#L14-L15